### ISSUE TITLE
Scroll wheel method deprecated in Pyside6?

### ISSUE THREAD

**Issue by MicahGale** [openmc-dev/plotter#146]
When I try to use the scroll wheel in plotter I get the following error on stderr:

``` python
Traceback (most recent call last):
  File "/home/mgale/mambaforge/lib/python3.10/site-packages/openmc_plotter/plotgui.py", line 342, in wheelEvent
    if event.delta() and event.modifiers() == QtCore.Qt.ShiftModifier:
AttributeError: 'PySide6.QtGui.QWheelEvent' object has no attribute 'delta'
```

I'm assuming `delta` was deprecated at some point.

Versions of libraries:

* openmc-plotter==0.4.0
* PySide6==6.7.0
* PySide6_Addons==6.7.0
* PySide6_Essentials==6.7.0
* qtconsole==5.4.3

This is a small slice, I figured you didn't want my full `pip freeze`.

**Comment by paulromano** [comment]
Already fixed in #134 and will be in next release

**Comment by MicahGale** [comment]
Awesome. Sorry for the extra issue.
